### PR TITLE
input output shapes added and formatting fixes

### PR DESCRIPTION
**Pull request by Roshrini**
@piiswrong 
- Input Output shape description for operators added
- Changes to show HybridLayer docs 
- formatting fixes

### CODE REVIEW

File: python/mxnet/foo/nn/conv_layers.py [hunk 1]
```diff
@@ -158,6 +158,22 @@ class Conv1D(_Conv):
         see Initializer.
     bias_initializer: Initializer for the bias vector
         see Initializer.
+
+
+    Input Shape:
```
Comment (piiswrong) [inline python/mxnet/foo/nn/conv_layers.py]: How would this render? Want want a new section.

Comment (Roshrini) [inline python/mxnet/foo/nn/conv_layers.py]: screen shot 2017-06-26 at 4 57 00 pm
This is how it looks

Comment (Roshrini) [inline python/mxnet/foo/nn/conv_layers.py]: screen shot 2017-06-26 at 5 02 06 pm
Complete screenshot

File: python/mxnet/foo/nn/conv_layers.py [hunk 2]
```diff
@@ -158,6 +158,22 @@ class Conv1D(_Conv):
...
+        This depends on the `layout` parameter. Input is 3D array of shape
+        (batch_size, channel, width) if `layout` is `NCW`.
```
Comment (piiswrong) [inline python/mxnet/foo/nn/conv_layers.py]: channel is not defined. In the parameters section these are in_filters and filters. we should mention filters/in_filters is the number of output/input channels somewhere

File: python/mxnet/foo/nn/conv_layers.py [hunk 3]
```diff
@@ -158,6 +158,22 @@ class Conv1D(_Conv):
...
+
+    Output Shape:
+        This depends on the `layout` parameter. Output is 3D array of shape
+        (batch_size, channel, out_width) if `layout` is `NCW`. out_width
```
Comment (piiswrong) [inline python/mxnet/foo/nn/conv_layers.py]: input and output channels are different.

File: python/mxnet/foo/nn/conv_layers.py (hunk 1 above)
Comment (piiswrong) [inline python/mxnet/foo/nn/conv_layers.py]: @kevinthesun Can we make these subtitles grey?

File: python/mxnet/foo/nn/conv_layers.py (hunk 2 above)
Comment (piiswrong) [inline python/mxnet/foo/nn/conv_layers.py]: or maybe just rename filters to channels.

Comment (Roshrini) [inline python/mxnet/foo/nn/conv_layers.py]: For conv1d,2d,3d operators, I made changes to differentiate input/output channels like in_channel(in_filters) and out_channel(filters). 
For pooling operators, can I just say channel(filters)?

**Pull request merged**